include/zephyr/net/wifi_mgmt.h

```diff
@@ -88,8 +88,10 @@ struct wifi_scan_result {
 	uint8_t ssid[WIFI_SSID_MAX_LEN];
 	uint8_t ssid_length;
 
+	uint8_t band;
```

Comment:
```suggestion
	enum wifi_frequency_bands band;
```
?

Reply:
Ah I guess, not since you can be on multiple bands.

Reply:
Also, in general we are using `uint8_t/unit32_t` for declaring enum fields in NCS, enums are only when assigning.